avalanche/training/supervised/l2p.py

```diff
@@ -0,0 +1,175 @@
+from typing import List, Optional
+
+import numpy as np
+import torch
+import torch.nn as nn
+from avalanche.training.plugins import EvaluationPlugin
+from avalanche.training.plugins.strategy_plugin import SupervisedPlugin
+from avalanche.training.plugins.evaluation import default_evaluator
+from avalanche.training.templates import SupervisedTemplate
+from avalanche.models.vit import create_model
+
+
+MODEL = "vit_base_patch16_224"
+
+
+class L2PTemplate(SupervisedTemplate):
+    """
+    Learning to Prompt (L2P) plugin.
```

Comment:
it's not a plugin, it's a strategy (I know that avalanche is a bit confusing on these differences).